pkg/cmd/pipeline/translate.go

```diff
@@ -43,19 +43,20 @@ import (
 )
 
 const (
-	nameField       = "name"
-	statusField     = "status"
-	outputField     = "output"
-	repoField       = "repository"
-	branchField     = "branch"
-	filenameField   = "filename"
-	yamlField       = "yaml"
-	iconField       = "icon"
-	actionLockField = "actionLock"
-	actionNameField = "actionName"
-	variablesField  = "variables"
-	devBranchField  = "dev-branch"
-	PhasesField     = "phases"
+	nameField        = "name"
+	statusField      = "status"
+	outputField      = "output"
+	repoField        = "repository"
+	branchField      = "branch"
+	filenameField    = "filename"
+	yamlField        = "yaml"
+	iconField        = "icon"
+	actionLockField  = "actionLock"
+	actionNameField  = "actionName"
+	variablesField   = "variables"
+	buildEnvVarField = "build-envs"
```

Comment:
Have you checked if some change is needed in the backend to avoid to lose this new fields in redeploys or similar?

Reply:
I tested and it seems to be working, also I couldn't find any reference where we might lose this